src/theme/components/link.ts

```diff
@@ -0,0 +1,42 @@
+const variants = {
+  link: {},
+  moreLink: {
+    alignItems: "center",
+    display: "inline-flex",
+    justifyContent: "space-between",
```

Comment:
I can't see why this is necessary.  I did some testing and unless you see where it is needed, this can be removed.

Reply:
You're right. I forgot why I needed this at some point but I'll remove.